### ISSUE TITLE
Accessibility

### ISSUE THREAD

**Issue by Nikoraito** [ViossaDiskordServer/ViossaDotNet#33]
Our site should have maximum accessibility in mind, especially for blind, colorblind, and deaf/HoH users.

Review each page and element to ensure the following measures are implemented as completely as possible:

Vision/keyboard accessibility:
- [ ] Use semantic html tags over divs and spans where possible
- [ ] Identify and correct situations where colorblindness reduces the navigability of the site, e.g. color palette collisions.
- [ ] Use `tabindex` and related attributes where necessary to help keyboard-navigability of the site
- [ ] Add recorded voice samples for Viossa snippets where tts is impossible

HoH accessibility:
- [ ] For links to audio-first resources like podcasts, ensure that existinge transcripts are also linked (e.g. on vikoli) for each such resource
